server/routes/routerRequestHandlers.test.ts

```diff
@@ -451,34 +435,6 @@ describe('routerRequestHandlers', () => {
       expect(next).toHaveBeenCalled()
     })
 
-    it('should return prisoners first name and last name without whitespace and a capital letter at the start', async () => {
```

Comment:
This test can come out entirely from `routerRequestHandler.test.ts` because its now the responsibility of the mapper itself, and this test exists in the mappers tests above ☝️